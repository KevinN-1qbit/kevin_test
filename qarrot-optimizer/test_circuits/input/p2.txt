Rotate -1: IIIIZ
Rotate 1: IIZII
Rotate 2: IIZIX
Rotate -2: IIZII
Rotate -2: IIIIX
Rotate 2: IZXII
Rotate -2: IZIII
Rotate -2: IIXII
Rotate 2: IXIIZ
Rotate -2: IIIIZ
Rotate -2: IXIII
Rotate 2: IIZII
Rotate 2: IIXII
Rotate 2: IIZII
Rotate 2: IIIZI
Rotate 2: IIIXI
Rotate 2: IIIZI
Rotate 1: IZIII
Rotate 1: IIIIZ
Rotate 1: IIIZI
Rotate 2: IXIIZ
Rotate -2: IIIIZ
Rotate -2: IXIII
Rotate 2: IIIZX
Rotate -2: IIIZI
Rotate -2: IIIIX
Rotate 2: IZIXI
Rotate -2: IZIII
Rotate -2: IIIXI
Rotate -1: IIIIZ
Rotate 2: IZIIX
Rotate -2: IZIII
Rotate -2: IIIIX
Rotate -1: IZIII
Rotate -1: IIIIZ
Rotate 1: IIIZI
Rotate 2: IIIZX
Rotate -2: IIIZI
Rotate -2: IIIIX
Rotate 2: IZIXI
Rotate -2: IZIII
Rotate -2: IIIXI
Rotate 2: IXIIZ
Rotate -2: IIIIZ
Rotate -2: IXIII
Rotate 2: IIIZI
Rotate 2: IIIXI
Rotate 2: IIIZI
Rotate 2: IIIZI
Rotate 2: IIIXI
Rotate 2: IIIZI
Rotate 1: ZIIII
Rotate 1: IZIII
Rotate 1: IIIZI
Rotate 2: XZIII
Rotate -2: IZIII
Rotate -2: XIIII
Rotate 2: IXIZI
Rotate -2: IIIZI
Rotate -2: IXIII
Rotate 2: ZIIXI
Rotate -2: ZIIII
Rotate -2: IIIXI
