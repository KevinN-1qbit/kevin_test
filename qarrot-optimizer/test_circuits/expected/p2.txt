Rotate -1: IIIIZ
Rotate 1: IIZII
Rotate 1: IIZIZ
Rotate -1: IIZXZ
Rotate -1: IZIII
Rotate -1: IZZXZ
Rotate 1: IZIXI
Rotate 1: ZIIII
Measure +: ZZZIZ
Measure +: IZZXZ
Measure +: IIXIX
Measure +: ZZZXZ
Measure +: IIZIZ
